.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,33 @@
+name: "CodeQL"
```

Please prepend `common-` to the file name to follow the `1x-` `2x-` `common-` convention.

---

Suggested more descriptive workflow name that also highlights the "common" part

```suggestion
name: "Common Static Check: CodeQL"
```

---

What's funny is that this came from THEIR template hehe

---

Seems that the top comment (`common-` file name prefix) has been overlooked.

Sadly, GH doesn't allow one to create two separate comment threads on the same line. That's how we ended up with 2 action intems in 1 thread.

---

Sorry, but just as a heads up, you can create two separate comment threads for one line

---

24a76c75ad2f5516a6532de60e0fad8783489b5d changes the file name :+1: